server/graphql/v2/query/ExpenseQuery.js

```diff
@@ -0,0 +1,23 @@
+import { GraphQLNonNull, GraphQLInt } from 'graphql';
+
+import { Expense } from '../object/Expense';
+import models from '../../../models';
+
+const ExpenseQuery = {
+  type: Expense,
+  args: {
+    id: {
+      /** TODO: When migration is done decode the id field in the resolver before querying the database.
+       * At the moment of creating this field the migration from v1 to v2 has just started and the id
+       * that is going to be used to query the expense is the unencoded one from the api v1.
+       */
```

Comment:
Then, the query should be able to resolve `id` in both forms from this day.

Either we support 2 different `args` for that, `id` / `internal_id`.

Or maybe smarter, we regexp match the `id` to check if it's in the new form. If it is, then we decode it. If it's not, we can check that the id is as integer. If it's not, we throw an error.

Reply:
Checking at the database I can see all ids are of type integer. I think checking if the string is a number or not would be enough here.